INSTALL.rst

```diff
@@ -219,10 +219,11 @@ glpk-cut-log (A fork of the GNU Linear Programming Kit)
 This can be used to speed up certain classes of problems for the arithmetic
 implementation in cvc5. (This is not recommended for most users.)
 
-glpk-cut-log can be installed using the ``contrib/get-glpk-cut-log`` script.
-Note that the only installation option is manual installation via this script.
-cvc5 is no longer compatible with the main GLPK library. Configure cvc5 with
-``configure.sh --glpk`` to build with this dependency.
+cvc5 is no longer compatible with the main GLPK library.
```

Comment:
```suggestion
cvc5 is not compatible with the official version of the GLPK library.
```
As far as I know it was never compatible with the official version to begin with.